Photo showcase

berkgaut opened the pull request:
This PR contains final updates to photosets and mostly concludes the pre-launch work

emeraldinna commented:
Boris, I noticed that you haven't filled in the 'alt' attribute. I want to highlight why it's recommended to use the 'alt' attribute in frontend development:

The HTML 'alt' attribute serves to provide alternative text for an image. This text becomes essential if the image cannot be displayed due to reasons such as a slow connection, an error in the 'src' attribute, or when a user relies on a screen reader. The text should describe the image if the image contains information.

berkgaut commented:
@emeraldinna I addressed your comment regarding alt attributes. This PR is now officially not a work in progress, so you can safely go ahead with the review
